diff --git a/calphy/alchemy.py b/calphy/alchemy.py
index d51e018..79e9fff 100644
--- a/calphy/alchemy.py
+++ b/calphy/alchemy.py
@@ -110,6 +110,7 @@ class Alchemy(cph.Phase):
         #check for melting
         self.dump_current_snapshot(lmp, "traj.equilibration_stage2.dat")
         self.check_if_melted(lmp, "traj.equilibration_stage2.dat")
+        self.dump_current_snapshot(lmp, "conf.equilibration.dump")
 
         #close object and process traj
         lmp = ph.write_data(lmp, "conf.equilibration.data")
diff --git a/calphy/input.py b/calphy/input.py
index 229bc21..f674c4e 100644
--- a/calphy/input.py
+++ b/calphy/input.py
@@ -251,6 +251,8 @@ class Calculation(InputTemplate):
             return True
         elif val in ["False", "false", 0, "0", False]:
             return False
+        elif val is None:
+            return False
         else:
             raise ValueError(f'Unknown bool input of type {type(val)} with value {val}')
 
diff --git a/calphy/liquid.py b/calphy/liquid.py
index a25901a..5c05f0f 100644
--- a/calphy/liquid.py
+++ b/calphy/liquid.py
@@ -145,6 +145,7 @@ class Liquid(cph.Phase):
         self.dump_current_snapshot(lmp, "traj.equilibration_stage1.dat")
         self.check_if_solidfied(lmp, "traj.equilibration_stage1.dat")
         self.dump_current_snapshot(lmp, "traj.equilibration_stage2.dat")
+        self.dump_current_snapshot(lmp, "conf.equilibration.dump")
         lmp = ph.write_data(lmp, "conf.equilibration.data")
         lmp.close()
 
diff --git a/calphy/solid.py b/calphy/solid.py
index 08e0a34..7ca5c8c 100644
--- a/calphy/solid.py
+++ b/calphy/solid.py
@@ -255,6 +255,7 @@ class Solid(cph.Phase):
         #check for melting
         self.dump_current_snapshot(lmp, "traj.equilibration_stage2.dat")
         self.check_if_melted(lmp, "traj.equilibration_stage2.dat")
+        self.dump_current_snapshot(lmp, "conf.equilibration.dump")
         lmp = ph.write_data(lmp, "conf.equilibration.data")
         #close object and process traj
         lmp.close()
@@ -328,6 +329,7 @@ class Solid(cph.Phase):
 
         #check for melting
         self.dump_current_snapshot(lmp, "traj.equilibration_stage2.dat")
+        self.dump_current_snapshot(lmp, "conf.equilibration.dump")
         lmp = ph.write_data(lmp, "conf.equilibration.data")
         #close object and process traj
 
